pkgs/top-level/all-packages.nix

```diff
@@ -5878,44 +5878,25 @@ in
   };
   purePackages = recurseIntoAttrs (callPackage ./pure-packages.nix {});
 
+  pythonNew = callPackage ./python-new-packages.nix {};
```

Comment:
unless `python` wouldn't point to the interpreter anymore, I think a better name is needed here. Any suggestions? Another possibility is to directly include the contents of `python-new-packages.nix` into all-packages.nix`. That way you have `pkgs.cpython27.pkgs`, `pkgs.pypy27.buildEnv`, etc.